lib/kontena/plugin/shell/session.rb

```diff
@@ -27,13 +27,20 @@ def run_command(buf)
         tokens = buf.split(/\s(?=(?:[^"]|"[^"]*")*$)/).map(&:strip)
         runner = Shell.command(tokens.first) || Shell.command(context.first) || Kontena::Plugin::Shell::KontenaCommand
         command = runner.new(context, tokens, self)
-        if fork_supported?
+        if fork_supported? && forkable_command?(command)
```

Comment:
I guess this implementation could have been in the kontena-command so then it wouldn't need this check.

Reply:
Like it was in #32

Reply:
I was too fast merger ... we can refactor this in a separate pr.